src/libcore/ops.rs

```diff
@@ -1901,6 +1958,35 @@ impl<'a, T: ?Sized> DerefMut for &'a mut T {
 }
 
 /// A version of the call operator that takes an immutable receiver.
+///
+/// # Examples
+///
+/// Closures automatically implement this trait, which allows them to be
+/// invoked. Note, however, that `Fn` takes an immutable reference to the
+/// argument. To take a mutable reference, implement [`FnMut`], and to consume
+/// the argument, implement [`FnOnce`].
```

Comment:
Here I would say "captures" instead of "argument" to avoid confusion (might need to define that word though).